Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml

```diff
@@ -115,20 +129,51 @@ script:
     - contextPath: Domain.RecordedFuture.LastSeen
       description: Risk last seen timestamp
       type: date
-    description: Returns threat intelligence information for a domain or DNS in Recorded Future.
-  - name: ip
-    arguments:
-    - name: ip
-      required: true
-      default: true
+    - contextPath: Domain.Tags
+      description: Tags that are associated with the Domain
+      type: String
+    - contextPath: Domain.CommunityNotes.Note
+      description: '...'
```

Comment:
A summary note on the mentioning of the domain in a source monitored by Recorded Future.